### PR TITLE
Add task solution

### PR DESCRIPTION
**Pull request by SerhiiUnhurian**
[DEMO LINK](https://SerhiiUnhurian.github.io/react_static-list-of-posts-js/)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,107 +1,29 @@
...
+import { PostList } from './components/PostList/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId)
+      || null;
+}
+
+function getCommentsById(commentId) {
+  return commentsFromServer.filter(comment => comment.postId === commentId)
+      || null;
```
Comment (etojeDenys) [inline src/App.jsx:15]: filter always return the array, so null is redundant here

**Review by etojeDenys — CHANGES_REQUESTED** [review]

**Review by etojeDenys — CHANGES_REQUESTED** [review]
also your demo link doesn't work, try to fix it

**Review by etojeDenys — APPROVED** [review]
lgtm

**Comment by SerhiiUnhurian** [comment]
Thanks)

**Pull request closed without merging**